.github/workflows/nostd.yml

```diff
@@ -1,10 +1,14 @@
+# This workflow checks whether the library is able to run without the std library (e.g. embedded).
+# This entire file should be removed if this crate does not support no-std
```

Comment:
Maybe also add a line here (and in the other non-`check` files) saying that all the shared configuration lines (like `on: push:`) are documented in `check.yml`, and then remove the repeated comments (like "If new code is pushed") from all the other files.